core/templates/dev/head/pages/interaction-specs.constants.ajs.ts

```diff
@@ -16,5 +16,7 @@
  * @fileoverview Constant file for the INTERACTION_SPECS constant.
  */
 
+const interactionSpecs = require('interactions/interaction_specs.json');
```

Comment:
Const would normally be named with all capital letters right? Not sure if it is necessary here.

Reply:
Yes, done.